data-android/src/main/java/app/tivi/data/TiviRoomDatabase.kt

```diff
@@ -62,7 +62,6 @@ import app.tivi.data.views.FollowedShowsWatchStats
     ],
     version = 26,
     autoMigrations = [
-        AutoMigration(from = 23, to = 24),
```

Comment:
No idea why the build doesn't like it. It's not a big issue to remove this anyway.